docs/player/gameboard.md

```diff
@@ -0,0 +1,81 @@
+
+### Grid
+
+The *grid* layer draws gridlines on the screen, on top of the *map* layer.
+At the moment, it only supports squares, but hexagonal maps are on the long-term to-do list.
+Players can only see the grid when the GM has turned grid visibility on, but cannot modify it themselves.
+
+### Tokens
+
+The *tokens* layer is above the grid layer and serves as the surface of the gaming board for the players.
+This is where your tokens are placed and where you can explore the map and/or fight monsters.
+Players can place shapes and move or otherwise modify shapes that they have sufficient access for.
+
+This is mostly done with the [select tool](/docs/tools/select/), but you can also [pan around](/docs/tools/pan/) or [zoom](/docs/tools/zoom/) to get a better view on the surroundings, occasionally [draw](/docs/tools/draw/) something or [ping](/docs/tools/ping) to highlight something on the gameboard for other players.
+Also, during combat encounters, the [initiative tool](/docs/tools/initiative) will be displayed free-floating on the gameboard.
+
+### DM-only layers
+There are two DM-only layers that players can neither directly see nor directly interact with.
+On the [DM](/docs/dm/layers/#dm) layer, the DM can position NPC, monsters, assets or everything else that can be positioned on a layer, for later use.
+This way, a band of bandits hidden in the woods or a group of invisible monsters does not need to be imported, placed and positioned *ad-hoc* when encountering them, but can be simply transferred to the *tokens* layer with some simple clicks.
+
+On the [FOW](/docs/dm/layers/#fow) layer (FOW translating to Fog of War), the DM will draw walls and other barriers that hinder movement and/or light to pass.
+
+## Floors
+
+All of the above layers combined make up a *floor*.
+This is to support vertically stacked places like dungeons or buildings.
+In this cases, every level of a dungeon or building could be represented by a floor.
+
+You can access floors at the menue at the bottom left of the game board.
+The selected floor will render on your game board, *above* all layers that are set below it.
+Note, though, that you might not be able to see everything of a layer, depending on the [lighting](/docs/dm/light-shadows) and whether you sufficient access to the tokens on this floor.
+
+As the floors are rendered on top of each others, you might be able to see the floors below the floor you are present on.
+A common example for this would be a window, balcony, or even a hole in the boarding.
+So if there is nothing hindering your sight, you can still see some things happening on the floors below (e.g. when looking out of a window).
+
+You are not able to change the floor you are on, yourself.
+Instead, you will be transferred to other floors by the DM.
+Making players able to change floors in certain places (where, e.g. stairs are on the map) is discussed bot not yet planned to be implemented in the forseeable future.
```

Comment:
This is not correct, players can change floors at will currently.  The DM can prevent some floors to be accessible by players since a recent change with the eye symbol though.

Reply:
I actually wanted to refer to player-controlled tokens, not the players themselves. Sorry & thanks for pointing that out. :)